src/utilities/fetchAndExtractZip.utils.ts

```diff
@@ -90,13 +115,14 @@ export const fetchAndUnpackZip = async (remoteTemplatesArchiveUrl: string): Prom
 			return { error: errorMessage, structure: undefined };
 		}
 
-		const zip = new JSZip();
-		const content = await zip.loadAsync(zipData);
-		const structure = await processZipContent(content);
+		const { structure } = await unpackFileZip(zipData);
 
 		return { structure };
 	} catch (error) {
-		const errorMessage = error instanceof Error ? `${error.name}: ${error.message}` : "Unknown error occurred";
+		const errorMessage =
+			error instanceof Error
+				? `${error.name}: ${error.message}`
+				: i18n.t("fetchAndExtract.uknownErrorUnpackingZip", { ns: "utilities" });
 		LoggerService.error(
 			namespaces.utilities.fetchAndExtract,
 			i18n.t("fetchAndExtract.fetchAndExtractError", {
```

Comment:
Missing file name and stack for the translation

Reply:
added

Reply:
cool